xoa_driver/functions/anlt_ll_debug.py

```diff
@@ -0,0 +1,126 @@
+from __future__ import annotations
+
+from typing import Dict
+from xoa_driver.enums import (
+    Layer1ConfigType
+)
+from xoa_driver.ports import GenericAnyPort
+from xoa_driver.lli import commands
+
+
+class anlt_ll_debug:
+    PMD_CONFIG_REGISTER = 0x02
+    LT_TX_CONFIG_REGISTER = 0x20
+    LT_TX_FRAME_REGISTER = 0x24
+    LT_RX_STATUS_REGISTER = 0x29
+    LT_RX_CONFIG_REGISTER = 0x28
+    LT_RX_FRAME_REGISTER = 0x2C
+    LT_RX_ERROR_STAT_0 = 0x2A
+    LT_RX_ERROR_STAT_1 = 0x2B
+
+    def __init__(self, port: GenericAnyPort, lane: int):
+        """
+        :param port: port to select
+        :type port: :class:`~xoa_driver.ports.GenericAnyPort`
+        :param lane: lane index, starting from 0. The lane to reset
+        :type lane: int
+        """
+        self.port = port
+        self.conn = port._conn
+        self.mid = port.kind.module_id
+        self.pid = port.kind.port_id
+        self.lane = lane
+
+    async def init(self) -> None:
+        ll_info = await commands.PL1_CFG_TMP(self.conn, self.mid, self.pid, self.lane, Layer1ConfigType.LL_DEBUG_INFO).get()
+        self.ll_info = {
+            "base": ll_info.value[0],
+            "rx_gtm_base": ll_info.value[1],
+            "rx_serdes": ll_info.value[2],
+            "tx_gtm_base": ll_info.value[3],
+            "tx_serdes": ll_info.value[4],
+        }
+
+    async def lane_reset(self) -> None:
+        """Reset the lane (serdes)"""
+        GTM_QUAD_GT_CONFIG = 0x102
+        addr = self.ll_info.get('rx_gtm_base') + GTM_QUAD_GT_CONFIG + (self.ll_info.get('rx_serdes') * 0x40)
```

Comment:
Can be a NoneType error, method `dict().get(<key_name>)` if not able to find an item by the provided key will return None as a default value, which can be a problem... 
Without knowing the use case example I only can recommend changing `self.ll_info` from a `dict` into a `dataclass` 

``` python
@dataclass
class LLInfo():
    base: int
    rx_gtm_base: int
    rx_serdes: int
    tx_gtm_base: int
    tx_serdes: int

...

async def init(self) -> None:
    ll_info = await commands.PL1_CFG_TMP(self.conn, self.mid, self.pid, self.lane, Layer1ConfigType.LL_DEBUG_INFO).get()
    self.ll_info = LLInfo(
        base=ll_info.value[0],
        rx_gtm_base=ll_info.value[1],
        rx_serdes=ll_info.value[2],
        tx_gtm_base=ll_info.value[3],
        tx_serdes=ll_info.value[4],
    )
    # Or an short hand but less explicit self.ll_info = LLInfo(*ll_info.value[:5])
```
then code will look like in the next example which will make the code more predictable and also use all advantages of IDE autocompletion
``` python
# addr = self.ll_info.get('rx_gtm_base') + GTM_QUAD_GT_CONFIG + (self.ll_info.get('rx_serdes') * 0x40)
addr = self.ll_info.rx_gtm_base + GTM_QUAD_GT_CONFIG + (self.ll_info.rx_serdes * 0x40)
```